src/lib/modules/Document/DocumentExtras.js

```diff
@@ -57,9 +57,7 @@ export class DocumentExtras extends Component {
           )}
         </>
       );
-    } else {
-      return 'No additional information.';
-    }
+    } else return null;
```

Comment:
I think you can rewrite that to something like `(!_isEmpty(metadata.alternative_titles) ||
      !_isEmpty(metadata.alternative_abstracts) )&& <>....</>`

Reply:
Thanks for bringing this up, i actually can remove this tab as the information is placed in different tabs